sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileScanRDD.scala

```diff
@@ -233,7 +233,7 @@ class FileScanRDD(
         if (files.hasNext) {
           currentFile = files.next()
           updateMetadataRow()
-          logInfo(s"Reading File $currentFile")
+          logInfo(log"Reading File ${MDC(PARTITIONED_FILE, currentFile)}")
```

Comment:
The class of `currentFile` is `PartitionedFile`.

Reply:
CURRENT_FILE seems better

Reply:
Done